__mocks__/react-native-fs-mock.js

```diff
@@ -0,0 +1,45 @@
+const rnfsMock = {
+  CachesDirectoryPath: jest.fn(),
+  DocumentDirectoryPath: jest.fn(),
+  ExternalDirectoryPath: jest.fn(),
+  ExternalStorageDirectoryPath: jest.fn(),
+  LibraryDirectoryPath: jest.fn(),
+  MainBundlePath: jest.fn(),
+  PicturesDirectoryPath: jest.fn(),
+  TemporaryDirectoryPath: jest.fn(),
```

Comment:
`XXXPath` members are not functions. They are just strings.

Reply:
Updated mock file with typings, missing methods, and mock primitive values with their intended values instead of mock fn